server/game/cards/01-Core/OneStoodAgainstMany.js

```diff
@@ -9,40 +9,18 @@ class OneStoodAgainstMany extends Card {
                 cardType: 'creature',
                 controller: 'self',
                 gameAction: ability.actions.sequential([
-                    ability.actions.resolveFight(context => ({
-                        attacker: context.target,
-                        promptForSelect: {
-                            activePromptTitle: 'Choose a creature to fight',
-                            cardType: 'creature',
-                            controller: 'opponent'
-                        },
-                        postHandler: (context, action) => this.chosenTargets = action.target
-                    })),
-                    ability.actions.resolveFight(context => ({
-                        attacker: context.target,
-                        promptForSelect: {
-                            activePromptTitle: 'Choose a creature to fight',
-                            cardType: 'creature',
-                            controller: 'opponent',
-                            cardCondition: card => !this.chosenTargets.includes(card)
-                        },
-                        postHandler: (context, action) => this.chosenTargets = this.chosenTargets.concat(action.target)
-                    })),
-                    ability.actions.resolveFight(context => ({
-                        attacker: context.target,
-                        promptForSelect: {
-                            activePromptTitle: 'Choose a creature to fight',
-                            cardType: 'creature',
-                            controller: 'opponent',
-                            cardCondition: card => !this.chosenTargets.includes(card)
-                        }
-                    }))
+                    ability.actions.ready(),
```

Comment:
Will this lose the "different targets" restriction?

Reply:
Ah shit, probably. I'll look at it when I get to the office.